pkg/core/prometheus.go

```diff
@@ -6,37 +6,46 @@ import (
 
 // Metrics for monitoring service.
 var (
-	//blockHeight prometheus metric.
+	// blockHeight prometheus metric.
 	blockHeight = prometheus.NewGauge(
 		prometheus.GaugeOpts{
 			Help:      "Current index of processed block",
 			Name:      "current_block_height",
 			Namespace: "neogo",
 		},
 	)
-	//persistedHeight prometheus metric.
+	// persistedHeight prometheus metric.
 	persistedHeight = prometheus.NewGauge(
 		prometheus.GaugeOpts{
 			Help:      "Current persisted block count",
 			Name:      "current_persisted_height",
 			Namespace: "neogo",
 		},
 	)
-	//headerHeight prometheus metric.
+	// headerHeight prometheus metric.
 	headerHeight = prometheus.NewGauge(
 		prometheus.GaugeOpts{
 			Help:      "Current header height",
 			Name:      "current_header_height",
 			Namespace: "neogo",
 		},
 	)
+	// mempoolUnsortedTx prometheus metric.
+	mempoolUnsortedTx = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Help:      "Mempool unsorted transactions",
+			Name:      "mempool_unsorted_tx",
```

Comment:
@roman-khimov, I've changed the name of this metric. I doubt anyone uses it in some scripts, but still, do you have any objections?

Reply:
IIUC you've changed the `help` string, but the `name` is the same. It's OK this way. Changing names is not OK, that's a part of the public node interface.

Reply:
Oh, you're right, somehow I thought that I've changed the name as far. So it's just a help, yes.